docs/monitoring/tag-support.md

```diff
@@ -3,18 +3,29 @@ description: Extract tags from metric sources, environment, assign metadata to h
 
 ## What's a Tag?
 
-To help you manage your metrics, hosts, and containers, and to help you
-create more useful dashboards, you can assign metadata to each
+To help you manage your metrics, hosts, and containers, and to help you create more useful dashboards, you can assign metadata to each
 host/server/container in the form of *tags*.
 
-Tags let you organize your hosts/servers/containers in different
-ways – for example by role, owner, or environment. Each tag consists of
-a key and a value, separated by the ':' character. Both key & value are
-case-sensitive.
+Tags let you organize your hosts/servers/containers in different ways – for example by role, owner, or environment. Each tag consists of
+a key and a value, separated by the ':' character. Both key & value are case-sensitive.
```

Comment:
We probably have to mention tags automatically added by agents based on monitored metric nature.